### ISSUE TITLE
Support android-arm64

### ISSUE THREAD

**Issue by Efreak** [speedyleion/gh-difftool#43]
Please support android-arm64 for future updayes. I have successfully built this and am using it, however it took around 20 minutes of downloading and building on my tablet, and it got rather warm; it would be nice if it could be installed through the cli instead; there are no changes necessary, and github actions should be able to perform the build.

**Comment by speedyleion** [comment]
This seems quite reasonable to be done during release similar to the other binaries.  
The only issue I see is i'm not sure how one is using gh cli on android. I feel like I would need to understand that to be able to fully support the builds if an issue arises running it.

To that end what method are you using to use the gh cli on android?

**Comment by Efreak** [comment]
I'm using it in [termux](https://termux.dev/en/) normally. GH CLI is available through the default termux repo:

```
❯ apt show gh
Package: gh
Version: 2.33.0
Maintainer: Krishna kanhaiya @kcubeterm
Installed-Size: 47.2 MB
Homepage: https://cli.github.com/
Download-Size: 7036 kB
APT-Manual-Installed: yes
APT-Sources: https://packages.termux.dev/apt/termux-main stable/main aarch64 Packages
Description: GitHub’s official command line tool


~  with 🐏 5GiB/7GiB | 2GiB/4GiB took 47ms
❯ which gh
/data/data/com.termux/files/usr/bin/gh

~  with 🐏 5GiB/7GiB | 2GiB/4GiB took 13ms
❯ gh repo list --limit 5

Showing 5 of 134 repositories in @Efreak                                                                                                                              Efreak/AI-Horde-Styles                 Styles to use to enhan...  public, fork  2h
Efreak/AI_Horde_Discord                A discord bot to inter...  public, fork  17h
Efreak/AI-Horde-image-model-reference                             public, fork  5d
Efreak/artbot-for-stable-diffusion     A front end GUI for in...  public, fork  19d
Efreak/hordelib                        A wrapper around Comfy...  public, fork  26d

~  with 🐏 5GiB/7GiB | 2GiB/4GiB took 842ms
❯
```

**Comment by speedyleion** [comment]
This should be available now via #46
